colcon_core/task/python/build.py

```diff
@@ -80,6 +80,12 @@ async def build(self, *, additional_hooks=None):  # noqa: D102
                 'build', '--build-base', os.path.join(
                     args.build_base, 'build'),
                 'install', '--prefix', args.install_base,
+                '--install-lib', str(get_python_install_path(
+                    'purelib', {'base': args.install_base})),
+                '--install-scripts', str(get_python_install_path(
+                    'scripts', {'base': args.install_base})),
```

Comment:
Will this conflict with the configured install scripts for using the libexec/ros2run pattern as here?

https://github.com/ros2/demos/blob/a1bc6dcbd71c12dceb33ee69e76b455b30f2177f/demo_nodes_py/setup.cfg#L1-L4

Bloom does [some nonsense](https://github.com/ros-infrastructure/bloom/blob/922a6d2208f1e19c0673582d149bc17093d9e225/bloom/generators/debian/generator.py#L395-L406) to only set install_scripts if it isn't already set for a package.

Reply:
The explicit options in `setup.cfg` seems to take precedence, but I realize now that there are more spots in `build.py` that would need to be updated, and I'm not sure all of the distutils commands support the necessary command line options. I'm no longer convinced this approach will pan out.